exchanges/bithumb/bithumb.go

```diff
@@ -428,6 +429,29 @@ func (b *Bithumb) PlaceTrade(orderCurrency, transactionType string, units float6
 	return response, nil
 }
 
+// ModifyTrade modifies an order already on the exchange books
+func (b *Bithumb) ModifyTrade(orderID, orderCurrency, transactionType string, units float64, price int64) (OrderPlace, error) {
+	response := OrderPlace{}
+
+	params := url.Values{}
+	params.Set("order_currency", common.StringToUpper(orderCurrency))
+	params.Set("Payment_currency", "KRW")
+	params.Set("type", common.StringToUpper(transactionType))
+	params.Set("units", strconv.FormatFloat(units, 'f', -1, 64))
+	params.Set("price", strconv.FormatInt(price, 10))
+	params.Set("order_id", orderID)
+
+	err := b.SendAuthenticatedHTTPRequest(privatePlaceTrade, params, &response)
+	if err != nil {
+		return response, err
+	}
+
+	if response.Status != noError {
```

Comment:
It looks like [this line](https://github.com/thrasher-/gocryptotrader/pull/222/files#diff-247f3ecc8045e689369053b156f2a642R673) removes the need to these checks anymore

Reply:
Will remove